hyp3_testing/templates/autorift_golden.json.j2

```diff
@@ -88,5 +88,45 @@
        ]
     },
     "job_type": "AUTORIFT"
+  },
+  {
+    "name": "{{ name }}",
+    "job_parameters": {
+      "granules": [
+        "LC09_L1GT_215109_20220125_20220125_02_T2",
+        "LC09_L1GT_215109_20220210_20220210_02_T2"
+       ]
+    },
+    "job_type": "AUTORIFT"
+  },
+  {
+    "name": "{{ name }}",
+    "job_parameters": {
+      "granules": [
+        "LE07_L1TP_063018_20040911_20200915_02_T1",
+        "LE07_L1TP_063018_20040810_20200915_02_T1"
+       ]
+    },
+    "job_type": "AUTORIFT"
+  },
+  {
+    "name": "{{ name }}",
+    "job_parameters": {
+      "granules": [
+        "LT05_L1GS_001013_19920425_20200915_02_T2",
+        "LT05_L1GS_001013_19920628_20200914_02_T2"
+       ]
+    },
+    "job_type": "AUTORIFT"
+  },
+  {
+    "name": "{{ name }}",
+    "job_parameters": {
+      "granules": [
+        "LT04_L1GS_050117_19890315_20200916_02_T2",
+        "LT04_L1GS_050117_19900129_20200916_02_T2"
```

Comment:
```suggestion
        "LT04_L1TP_063018_19880611_20200917_02_T1",
        "LT04_L1TP_063018_19880627_20200917_02_T1"
```
This pair ran fine in the datacube test. I've run it again in production now that hyp3-autorift is released:
https://hyp3-api.asf.alaska.edu/jobs/879efe81-adf2-48cb-89db-f4bd88c577e4